kubernetes/istio-install/istio-ca.yaml

```diff
@@ -20,3 +20,4 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
+---
```

Comment:
Not sure what you mean :-( - but I guess removing/moving it is independent of this PR.